recipes-ni/initscripts-nilrt/initscripts-nilrt_1.0.bb

```diff
@@ -85,6 +87,7 @@ do_install () {
 	update-rc.d -r ${D} nisetupkernelconfig   start 3  4 5 .
 	update-rc.d -r ${D} nisetcommitratio      start 99 S .
 	update-rc.d -r ${D} wirelesssetdomain     start 36 S .
+	update-rc.d -r ${D} cleanvarcache         start 38 0 1 6 S .
```

Comment:
I agree, this script seems like it could be specified as `start 38 S .` similar to how `nicleanstalelinks` is.

Reply:
Shouldn't we keep 1 and 6 also, in case next boot is safemode?

Reply:
We enter runlevel `6` when issuing a `reboot`, so if we want to clear /var/cache prior to reboot then yeah, adding it to `6` start could be appropriate.

Where would we enter runlevel `1`?

Reply:
Sorry I meant 0 not 1

Reply:
Oh, yeah, in that case, I agree-- if we clean up on shutdown (6) we should do the same on halt (0).

Reply:
I'll remove 1. I've just thought about another scenario that isn't handled. When there's a power outage of some sort and the target reboots into safe mode, cleanup doesn't happen until the next time the target reboots into runmode..

Reply:
Isn't initscripts-nilrt_1.0.bb applicable to both safemode and runmode images ([this](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-core/packagegroups/packagegroup-ni-base.bb#L98), [this](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-core/images/niconsole-image-safe.bb#L4) [this](https://github.com/ni/meta-nilrt/blob/nilrt/master/sumo/recipes-core/images/niconsole-image.inc#L16) suggest it is)? - in which case cleanup will happen during boot in either mode .. and my previous comment about needing 1 and 6 isn't accurate since we have S (but I prefer we clean up in 1 and 6, S is optional).

Reply:
Right, you are. I verified that safemode image had the script too.